src/UnderlineNav2/UnderlineNav.test.tsx

```diff
@@ -115,12 +113,7 @@ describe('UnderlineNav', () => {
     await user.click(item)
     expect(onSelect).toHaveBeenCalledTimes(1)
 
-    if (REACT_VERSION_LATEST) {
-      expect(spy).not.toHaveBeenCalled()
-    } else {
-      // Warning: It looks like you're using the wrong act() around your test interactions
-      expect(spy).toHaveBeenCalledTimes(2)
-    }
+    expect(spy).not.toHaveBeenCalled()
```

Comment:
Can we remove the spy as well or do we need to make sure and test that the warning doesn't render? If the latter, when would be a good time to remove it all together? 👀 

I am not against to keeping it btw, just trying to understand 😅

Reply:
@broccolinisoup I think you're totally right that we can just remove it! 👍